### PR TITLE
#15 Feat: Added Footer Component

### PR DESCRIPTION
**Pull request by clever-cr**
### Task Description _(❗️Required)_
Added footer section for desktop, tablet and mobile breakpoints
### Type of Task

- [ ] fix
- [ ] chore
- [x] feat
- [ ] refactor
- [ ] perf
- [ ] docs

### Guidelines

- [x] Does your code follow ES6+ syntax?
- [x] Did you use arrow functional components?
- [x] Did you thoroughly review your code?
- [ ] Have you checked if your code does not introduce conflicts in the main codebase?

### Ticket Link _(❗️Required)_
https://www.notion.so/apeunit/9c987d9f0f6349ffb7f08c96a12192a3?v=41a7f9534b344a5580c0aae19224d313&p=d71fd22c40f448838b6c66b26c636ca3&pm=s

### CODE REVIEW

File: components/Layout/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+          <p>Ape Unit GmbH</p>
+          <p>
+            Waldemarstraße 38,<br></br> 10999 Berlin
+          </p>
+        </div>
+        <div>
+          <LinkItem to={"https://twitter.com/apeunit"}>Twitter</LinkItem>
+
+          <LinkItem to={"https://www.facebook.com/apeunit/"}>Facebook</LinkItem>
+          <LinkItem to={"https://de.linkedin.com/company/ape-unit"}>
```
Comment (mercihabam) [inline components/Layout/Footer.jsx]: Don't wrap string inside curly braces, you can replace the `to={""}` with `to=""`

**Review by mercihabam — CHANGES_REQUESTED** [review]
## You did well

Just small changes to apply.

File: components/Layout/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import Wrapper from "../sections/Wrapper";
+
+function Footer() {
+  return (
+    <Wrapper>
+      <div className="flex text-2xl font-neuf space-y-12 lg:space-y-0 lg:justify-between  flex-col lg:flex-row pr-91 pb-12.5 mt-31">
+        <div>
+          <p>Ape Unit GmbH</p>
+          <p>
+            Waldemarstraße 38,<br></br> 10999 Berlin
```
Comment (Odile078) [inline components/Layout/Footer.jsx]: instead of using `<br></br>` use `<br/>`

File: components/Layout/Footer.jsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+          </p>
+        </div>
+        <div>
+          <LinkItem to="https://twitter.com/apeunit">Twitter</LinkItem>
+
```
Comment (alain-kubwayo) [inline components/Layout/Footer.jsx]: Remove this unnecessarily empty line to make your code more readable.

**Review by alain-kubwayo — CHANGES_REQUESTED** [review]
Good job!
```
See the comment

**Review by alain-kubwayo — DISMISSED** [review]
Looks good!

**Review by Odile078 — DISMISSED** [review]

**Review by Lily-Lilian — DISMISSED** [review]

**Review by alain-kubwayo — DISMISSED** [review]
Looks good! 🚀

**Review by Lily-Lilian — DISMISSED** [review]
LGTM!!

**Review by mercihabam — DISMISSED** [review]

**Review by Odile078 — DISMISSED** [review]

**Review by Lily-Lilian — APPROVED** [review]

**Review by Odile078 — APPROVED** [review]

**Review by mercihabam — APPROVED** [review]

**Pull request merged**